web/static/js/pages/vehicle-update.js

```diff
@@ -32,47 +77,52 @@ spa.onNavigate('vehicle-update', (page, params) => {
 });
 
 
- //Cleaning listernes from this page
+ // *Cleaning listernes from this page:
 spa.onUnload('vehicle-update', (page) => {
+   // *Cleaning the event submit:
    $('#vehicle-update-form').off('submit');
+   // *Cleaning the event change:
    $('#vehicle-update-pic').off('change');
 });
```

Comment:
The system should clean the inputs when the user left this page (this includes all the text fields, file fields, and the vehicle photo thumbnail)